### PR TITLE
setcap: add page

### PR DESCRIPTION
**Pull request by RaviRahar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/setcap.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# setcap
+
+> Set capabilities of specified file.
+> Also see getcap, accessible via `tldr getcap`.
```
Comment (kbdharun) [inline pages/linux/setcap.md:4]:
```suggestion
> See also: `tldr getcap`.
```

File: pages/linux/setcap.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+> More information: <https://manned.org/setcap>.
+
+- Set capability cap_net_raw (to use RAW and PACKET sockets) for a given file:
```
Comment (kbdharun) [inline pages/linux/setcap.md:7]:
```suggestion
- Set capability `cap_net_raw` (to use RAW and PACKET sockets) for a given file:
```

File: pages/linux/setcap.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+`setcap 'cap_net_raw' path/to/file`
```
Comment (kbdharun) [inline pages/linux/setcap.md:9]:
```suggestion
`setcap '{{cap_net_raw}}' {{path/to/file}}`
```

File: pages/linux/setcap.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+- Set multiple capabilities on a file (ep behind the capability means "effective permitted"):
+
+`setcap 'cap_dac_read_search,cap_sys_tty_config+ep' path/to/file`
```
Comment (kbdharun) [inline pages/linux/setcap.md:13]:
```suggestion
`setcap '{{cap_dac_read_search,cap_sys_tty_config+ep}}' {{path/to/file}}`
```

File: pages/linux/setcap.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+- Remove all capabilities from a file:
+
+`setcap -r path/to/file`
```
Comment (kbdharun) [inline pages/linux/setcap.md:17]:
```suggestion
`setcap -r {{path/to/file}}`
```

File: pages/linux/setcap.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+- Verify that the specified capabilities are currently associated with the specified file:
+
+`setcap -v 'cap_net_raw' path/to/file`
```
Comment (kbdharun) [inline pages/linux/setcap.md:21]:
```suggestion
`setcap -v '{{cap_net_raw}}' {{path/to/file}}`
```

File: pages/linux/setcap.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+- The optional -n <rootuid> argument can be used to set the file capability for use only in a user namespace with this root user ID owner:
```
Comment (kbdharun) [inline pages/linux/setcap.md:23]:
```suggestion
- The optional `-n {{rootuid}}` argument can be used to set the file capability for use only in a user namespace with this root user ID owner:
```

File: pages/linux/setcap.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+`setcap -n <rootuid> 'cap_net_admin' path/to/file`
```
Comment (kbdharun) [inline pages/linux/setcap.md:25]:
```suggestion
`setcap -n {{rootuid}} '{{cap_net_admin}}' {{path/to/file}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

Note: We use `See also:` to reference pages in header/command descriptions, we use `{{}}` to highlight placeholder in our clients.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by einverne — APPROVED** [review]
LGTM.

File: pages/linux/setcap.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# setcap
+
+> Set capabilities of specified file.
+> See also: `tldr getcap`.
+> More information: <https://manned.org/setcap>.
+
+- Set capability `cap_net_raw` (to use RAW and PACKET sockets) for a given file:
+
+`setcap '{{cap_net_raw}}' {{path/to/file}}`
```
Comment (cyqsimon) [inline pages/linux/setcap.md:9]: The syntax given here is incorrect no?

At least as of time of writing (and it hasn't changed to the best of my knowledge), `setcap` has always required an operator to be specified for each clause. See [`man cap_text_formats`](https://manned.org/man/cap_text_formats).

Please someone check that I'm not crazy.

Comment (cyqsimon) [reply pages/linux/setcap.md:9]: `cap_text_formats` was only recently created (split off from [`man cap_from_text`](https://manned.org/man/debian-trixie/cap_from_text)). FYI for anyone interested in exploring the page's history.

**Pull request merged**